src/lib/singularity.h

```diff
@@ -158,14 +158,4 @@
     extern int singularity_fork_exec(char **argv);
 
 
-
-
-    extern int singularity_config_open(char *config_path);
-    extern void singularity_config_close(void);
-    extern void singularity_config_rewind(void);
-
-    extern char *singularity_config_get_value(char *key);
-    extern int singularity_config_get_bool(char *key, int def);
-
```

Comment:
Is there a reason we don't do the same definitions and declarations here as we do in `config_parser.h`? It's currently this way for all header files that implement `singularity_*` functions (singularity.h includes all `singularity_*` functions)